mne/preprocessing/artifact_detection.py

```diff
@@ -333,3 +334,189 @@ def _annotations_from_mask(times, art_mask, art_name):
             durations.append(times[l_idx[-1]] - times[l_idx[0]])
         desc.append(art_name)
     return Annotations(onsets, durations, desc)
+
+
+@verbose
+def annotate_breaks(raw, events=None, *,
```

Comment:
I don't see much value in the `*` here, I would just put it before `verbose` to make it safer for if/when we later add new kwargs